tests/TestCase.php

```diff
@@ -91,91 +93,41 @@ protected function dummyConnections()
         ];
     }
 
-
-    protected function dummyListing()
+    protected function dummyListingNewVersion()
```

Comment:
You don't need to rename it because you're replacing it instead of keeping both.